app/controllers/public/schedule.js

```diff
@@ -9,4 +9,43 @@ export default class ScheduleController extends Controller {
   get isSchedulePublished() {
     return this.model.event.schedulePublishedOn.toISOString() !== moment(0).toISOString();
   }
+
+   hideSchedule = true;
+   hideTrack = false;
+   hideRooms = false;
+  @action
+  showSchedule() {
+    this.toggleProperty('hideSchedule');
+    this.toggleProperty('hideTrack');
```

Comment:
Codacy found an issue: [Expected indentation of 5 spaces but found 4.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=6077057)